modules/custom/openy_hours_formatter/templates/openy-hours-formatter.html.twig

```diff
@@ -14,12 +14,12 @@
 {% if hours %}
 <div class="today-hours">
   {{ hours }}
-  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false" id="dropdownBranchHours" aria-haspopup="true">
+  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false" id="dropdownBranchHours-1" aria-haspopup="true">
```

Comment:
Nope
You should use field delta here

Reply:
No matter if you have 2+ html nodes with id=test or id=test1 or id=gggggg1233437483
You can't have 2 equal ids on a page per HTML standard

Reply:
@podarok Could you review it? https://github.com/ymcatwincities/openy/pull/1047/commits/02a9f57cab9be27afaeca42cc8d384976401765e